gateway/engine/beans/src/main/java/io/apiman/gateway/engine/beans/util/CaseInsensitiveStringMultiMap.java

```diff
@@ -456,5 +456,10 @@ public int getByte(String input, long offset) {
         public ByteOrder byteOrder(String input) {
             return ByteOrder.nativeOrder();
         }
+
+        @Override
+        protected Access<String> reverseAccess() {
+            return null;
```

Comment:
Why have you left this null?

Reply:
Was just a test if it builds at all with adding that.

Reply:
Oh I just noticed that it is not a draft PR (I thought i pressed the button)